File: timecapsule-api/src/main/java/com/timecapsule/api/TimecapsuleApiApplication.java
package com.timecapsule.api;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;

@SpringBootApplication(exclude={DataSourceAutoConfiguration.class})
public class TimecapsuleApiApplication {

	public static void main(String[] args) {
		SpringApplication.run(TimecapsuleApiApplication.class, args);
	}

}

File: timecapsule-api/src/main/kotlin/com/timecapsule/api/TimecapsuleApiApplication.kt
package com.timecapsule.api

import com.timecapsule.database.config.DatabaseConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Import

@Import(DatabaseConfig::class)
@SpringBootApplication
class TimecapsuleApiApplication

fun main(args: Array<String>) {
    runApplication<TimecapsuleApiApplication>(*args)
}
